src/lib/viewers/text/PlainTextViewer.js

```diff
@@ -113,8 +113,8 @@ class PlainTextViewer extends TextBaseViewer {
         super.setup();
 
         this.textEl = this.createViewer(document.createElement('pre'));
-        this.textEl.className = 'bp-text bp-text-plain hljs';
-        this.textEl.classList.add(CLASS_HIDDEN);
+        this.textEl.classList.add('bp-text', 'bp-text-plain', 'hjls', CLASS_BOX_PREVIEW_CUSTOM_FOCUS, CLASS_HIDDEN);
```

Comment:
I don't think multiple arguments to `classList.add` are supported by IE11. You could use the `classnames` helper, instead?

https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility